src/BaselineOfSQLite3/BaselineOfSQLite3.class.st

```diff
@@ -28,11 +28,14 @@ BaselineOfSQLite3 >> baseline: spec [
 				package: 'SQLite3-Glorp' with: [ spec requires: #('Core' 'Glorp-Core') ];
 				group: 'glorp' with: 'SQLite3-Glorp';
 				
+				package: 'SQLite3-Inspector-Extensions' with: [ spec requires: #('Core') ];
```

Comment:
Are the Tools working for all the supported Pharo versions? If not, we need to put in the specific config inside `versionSpecificBaseline: ` method

Reply:
Done in https://github.com/pharo-rdbms/Pharo-SQLite3/pull/49/commits/5bac78d445b78ef423731ca1df66c9ac12d3b2c9